src/test/resources/data/InstructorStudentListPageUiTest.json

```diff
@@ -144,10 +144,10 @@
     },
      "Student3Course2": {
       "googleId": "",
-      "email": "barney.c.tmms@gmail.tmt",
+      "email": "hugh.c.tmms@gmail.tmt",
```

Comment:
Oh, I noticed this problem, but I'm not sure whether should I change it or not, Since 'Student4Course2' is Ted Mosby(ted.c.tmms@gmail.tmt) and 'Student5Course2' is Jack Lawrence(jack.c.tmms@gmail.tmt)....Should change these two to ted.m and jack.l respectively ?